tests/unit/handle_trailer_test.sh

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env bash
+
+include './src/handle_trailer.sh'
+include './tests/unit/utils.sh'
+
+# Correct trailers for --add-reviewed-by and -r
+CORRECT_REVIEWED_COMMIT="Signed-off-by: kw <kw@kw>
+Reviewed-by: John Doe <johndoe@community.com>
+
```

Comment:
Why do we need these extra spaces (for all the options), and why is the signed-off-by duplicate? This does not look correct.

Reply:
I believe the confusion here happened because of our poor choice of names for the variables and lack of better comments to explain what these variables are holding. Those variables are holding the trailer lines of multiple commits. For instance, when using the command `git local --max-count 2 --format="%(trailers)"` the output follows this format:

```
Signed-off-by: Name <mail>                              <- Most recent commit
Reviewed-by: Another Name <another mail>
                                                        <- Empty line splitting trailers from different commits
Signed-off-by: Name <mail>                              <- Second most recent commit trailers
Reviewed-by: Some Name <some mail>
```

This is why the `Signed-off` appears duplicated in multiple outputs and there are these extra spaces in all the options. We hope this clarified the issue. We've added some comments and changed the name of some of these variables to make it easier to understand.

Also please tell us if we misunderstood the problem here. Perhaps we could be missing something important

Reply:
I believe the confusion here happened because of our poor choice of names for the variables and lack of better comments to explain what these variables are holding. Those variables are holding the trailer lines of multiple commits. For instance, when using the command `git log --max-count 2 --format="%(trailers)"` the output follows this format:

```
Signed-off-by: Name <mail>                  <- Most recent commit
Reviewed-by: Another Name <another mail>
                                            <- Empty line splitting trailers from different commits
Signed-off-by: Name <mail>                  <- Second most recent commit trailers
Reviewed-by: Some Name <some mail>
```

This is why `Signed-off` appears duplicated in multiple outputs and there are these extra spaces in all the options. We hope this clarified the issue. We've added some comments and changed the name of some of these variables to make it easier to understand.

Also please tell us if we misunderstood the problem here. Perhaps we could be missing something important